src/Scaffolding/VS.Web.CG.Mvc/Templates/Blazor/Create.tt

```diff
@@ -44,10 +44,11 @@
                     string propertyShortTypeName = property.ShortTypeName.Replace("?", string.Empty);
                     var inputTypeName = Model.GetInputType(propertyShortTypeName);
                     var inputClass = Model.GetInputClassType(propertyShortTypeName);
+                    var requiredHtml = property.IsRequired ? "aria-required=\"true\"" : string.Empty;
```

Comment:
nit: might name this requiredAttribute or something rather than requiredHtml just to be a little more specific on what it is? Same on other files.

Reply:
sure yeah